### PR TITLE
OS X application changes

### LINKED ISSUE #266: OS X Application changes
**Branch:** `develop` (os x installers) 

Since we are offloading some functionalities to the installer, the .app must be changed for such.

REF: [OS X Installer changes](https://trello.com/c/YrrNklzn/85-os-x-installer-changes)

Since we allow the customization of paths for KA-Lite (database, contents, etc), the .app must also be updated to set the environment variables.

- [x] remove pyrun-2.7
- [x] remove assessment.zip
- [x] remove copying the bin/kalite executable
- [x] remove reset app
- [x] remove the shebangcheck.py
- [x] remove the setup of environment variables like KALITE_PYTHON, etc
- [x] remove making the `/Library/LaunchAgents/org.learningequality.kalite.plist`

### PR DESCRIPTION
**Pull request by amodia**
@cpauya this fixes #266

### CODE REVIEW

File: osx/KA-Lite/KA-Lite/AppDelegate.m [hunk 1]
```diff
@@ -24,37 +24,6 @@ @implementation AppDelegate
...
-    
-    if (pathExists(assessmentPath)) {
-        [self runKalite:command];
-        showNotification(@"Assessment items successfully extracted.");
-    } else {
-        NSLog([NSString stringWithFormat:@"Assessment file not found"]);
-    }
-}
-
 //REF http://objcolumnist.com/2009/08/09/reopening-an-applications-main-window-by-clicking-the-dock-icon/
```
Comment (cpauya) [inline osx/KA-Lite/KA-Lite/AppDelegate.m]: Change this to "// REF:" format.

File: osx/KA-Lite/KA-Lite/AppDelegate.m [hunk 2]
```diff
@@ -197,22 +170,14 @@ - (void) runTask:(NSString *)command {
...
-    // Set KALITE_PYTHON environment
-    [kaliteEnv addEntriesFromDictionary:[[NSProcessInfo processInfo] environment]];
-    [kaliteEnv setObject:pyrun forKey:@"KALITE_PYTHON"];
- 
     NSTask* task = [[NSTask alloc] init];
     NSString *kaliteCommand = [NSString stringWithFormat:@"%@ %@", kalitePath, command];
     NSArray *array = [kaliteCommand componentsSeparatedByString:@" "];
     
-    [task setEnvironment: kaliteEnv];
     [task setLaunchPath: pyrun];
```
Comment (cpauya) [inline osx/KA-Lite/KA-Lite/AppDelegate.m]: Why do we still need `pyrun` here?

**Comment by amodia** [comment]
We still need `pyrun` for now, but I will remove that later and replace it with kalite executable.

**Comment by djallado** [comment]
@amodia, I tested your PR. I can run it in Xcode but I need `0.15.x` build so that I can `start` and `open in browser`. here's the screenshots
screen shot 2015-11-12 at 9 00 03 pm
.
screen shot 2015-11-12 at 8 58 45 pm

**Comment by cpauya** [comment]
All good then!  Thanks @djallado for testing.  Merging this so we can proceed.

**Pull request merged**